run_iwyu_tests.py

```diff
@@ -124,8 +129,9 @@ def PrintLoadedTestsAndFiles():
   (runner_args, _) = parser.parse_known_args(unittest_args)
 
   if runner_args.run_test_file:
```

Comment:
This whole block is getting complicated enough that I think it might make sense to extract it into a `RunTestFile` function, something like:
```
if runner_args.run_test_file:
  exit(RunTestFile(runner_args.run_test_file))
```

to mirror the existing `PrintLoadedTests...` functions

Reply:
Thanks! I'm stealing this and retrofitting on current master; if you can rebase on that, it will be way clearer what's being added here. Sorry about the back and forth. I'll ping back when I've tested it.

Reply:
Now on master, please rebase.